src/components/dataset/query/sidebar/panels/DatasetAccess.jsx

```diff
@@ -0,0 +1,105 @@
+  static propTypes = {
+    classes: PropTypes.object,
+    helpOverlayToggle: PropTypes.func,
+    policies: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  static helpTitle = (
+    <Typography variant="h4">Roles and memberships in Terra Data Repository</Typography>
+  );
+
+  static helpContent = (
+    <Fragment>
+      <Typography variant="h6">Stewards</Typography>
+      <Typography>
+        Creating a dataset makes a user the Steward (or owner) of it. The steward of a dataset can:
+        <ul>
+          <li>Define the schema for a dataset</li>
+          <li>Delete a dataset (need to delete any resources leveraging the dataset first)</li>
+          <li>Assign new stewards, custodians and snapshot creators of a dataset</li>
+          <li>Add file/metadata to a dataset</li>
+          <li>Soft delete data (a function that hides data rather than permanently deletes it)</li>
+          <li>Read all the data in a dataset</li>
+          <li>Create a snapshot from a dataset</li>
+          <li>List all the stewards, custodians and snapshot creators of a dataset</li>
+        </ul>
+      </Typography>
+      <Typography variant="h6">Custodians</Typography>
+      <Typography>
+        A Steward can assign a Custodian to a dataset. The custodian of a dataset can:
+        <ul>
+          <li>Add file/metadata to a dataset</li>
+          <li>Soft delete data (a function that hides data rather than permanently deletes it)</li>
+          <li>Read all the data in a dataset</li>
+          <li>Create a snapshot from a dataset</li>
+          <li>List all the stewards, custodians and snapshot creators of a dataset</li>
+        </ul>
+      </Typography>
+      <Typography variant="h6">Snapshot Creator</Typography>
+      <Typography>
+        A Steward can assign a Snapshot Creator to a dataset. The Snapshot Creator of a dataset can:
+        <ul>
+          <li>Read all the data in a dataset</li>
+          <li>Create a snapshot from a dataset</li>
+          <li>List all the stewards, custodians and snapshot creators of a dataset</li>
+        </ul>
+      </Typography>
+    </Fragment>
+  );
+
+  render() {
+    const { classes, helpOverlayToggle, policies } = this.props;
+    const helpOverlayToggleWithContent = () =>
+      helpOverlayToggle(DatasetAccess.helpTitle, DatasetAccess.helpContent);
+
+    const custodiansObj = policies.find((policy) => policy.name === 'custodian');
+    const custodians = (custodiansObj && custodiansObj.members) || [];
+    const snapshotCreatorsObj = policies.find((policy) => policy.name === 'snapshot_creator');
+    const snapshotCreators = (snapshotCreatorsObj && snapshotCreatorsObj.members) || [];
+    const stewardsObj = policies.find((policy) => policy.name === 'steward');
+    const stewards = (stewardsObj && stewardsObj.members) || [];
+
+    return (
+      <div className={classes.root}>
+        <div>
+          <Link href="#" className={classes.blueLink} onClick={helpOverlayToggleWithContent}>
```

Comment:
The `#` might mess up our routing.  It's cleaner to to handle this with an action.  So there would be an action that triggers saving the 'is expanded' state in the central store and then the component can just pull that information from that store.

I don't love that we're using component state and not the main store (since now we have state scattered across different areas)